habitat_sim/simulator.py

```diff
@@ -297,14 +298,23 @@ def get_observation(self):
             if self._spec.gpu2gpu_transfer:
                 return self._gl_tensor.tensor().flip(0).squeeze(-1).clone()
             else:
+                size = self.__sensor_object.framebuffer_size
```

Comment:
```suggestion
                size = self._sensor_object.framebuffer_size
```

Looks like an additional underscore got added here (my bad!)

Reply:
Uh oh. Was tired and didn't spot the error either. Should have tested before pushing.